seller/src/main/java/com/sample/ecommerce/seller/main/Seller.java

```diff
@@ -0,0 +1,18 @@
+package com.sample.ecommerce.seller.main;
+
+
+import org.elasticsearch.common.joda.time.DateTime;
+
+import lombok.Data;
+
+/**
+ * Created by I307690 on 06-Dec-15.
+ */
+@Data
+public class Seller {
+  String sellerId;
+  String FirstName;
+  String LastName;
+  DateTime createdAt;

```

Comment:
you can initialize the DateTime here as well. Use JodaTime directly, not elasticsearch.